paymentwall/widget.py

```diff
@@ -11,14 +11,15 @@
 
 class Widget(Paymentwall):
 
-	BASE_URL = 'https://api.paymentwall.com/api'
-
 	def __init__(self, user_id, widget_code, products=[], extra_params={}):
 		self.user_id = user_id
 		self.widget_code = widget_code
 		self.extra_params = extra_params
 		self.products = products
 
+	def get_base_url(self):
```

Comment:
It's better to have 2 const `BASE_PW_URL` and `BASE_PW_API_URL` and have this method returns the url based on api type